nibabel/minc1.py

```diff
@@ -269,6 +269,7 @@ class MincHeader(Header):
     """
     # We don't use the data layout - this just in case we do later
     data_layout = 'C'
+    sizeof_hdr = 4
```

Comment:
`sizeof_hdr` seems a bit confusing.  Means size of needed sniff, but not the size of the header.